plugin/src/templates/edge/ipx.ts

```diff
@@ -1,5 +1,5 @@
 import { Accepts } from 'https://deno.land/x/accepts@2.1.1/mod.ts'
-import type { Context } from 'netlify:edge'
+import type { Context } from 'https://edge.netlify.com'
```

Comment:
For folks that are using the current plugin, will `netlify:edge` will still work?

Reply:
Let me verify!

Reply:
@nickytonline yes it should still work :)